classroom.rb

```diff
@@ -0,0 +1,12 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_classroom(student)
+    @students.push(student)
+  end
+end
```

Comment:
Since our aim is  to ensure a classroom has many students, I'd suggest you refactor the method to look similar to this:
```ruby
 def add_student(student)
    @students << student
    student.classroom = self
  end
```